app/uk/gov/hmrc/nisp/controllers/AccountController.scala

```diff
@@ -67,20 +84,19 @@ trait AccountController extends NispFrontendController with AuthorisedForNisp wi
           metricsService.abTest(getABTest(nino, spSummary.contractedOutFlag))
 
           customAuditConnector.sendEvent(AccountAccessEvent(nino, spSummary.contextMessage,
-            spSummary.statePensionAge.date, spSummary.statePensionAmount.week, spSummary.forecastAmount.week,
-            spSummary.dateOfBirth, user.name, spSummary.contractedOutFlag, spSummary.forecastOnlyFlag,
-            getABTest(nino, spSummary.contractedOutFlag), spSummary.copeAmount.week, authenticationProvider))
+            spSummary.statePensionAge.date, spSummary.statePensionAmount.week, spSummary.forecast.forecastAmount.week,
+            spSummary.dateOfBirth, user.name, spSummary.contractedOutFlag, spSummary.forecast.scenario, getABTest(nino, spSummary.contractedOutFlag), spSummary.copeAmount.week, authenticationProvider))
```

Comment:
Why was getABTest removed from here? The views shouldn't be calling any controller methods.